### PR TITLE
alexa change caption for maricay

### PR DESCRIPTION
**Pull request by maricay**
test comments

### CODE REVIEW

File: _posts/0000-01-21-maricay.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![fry](https://cloud.githubusercontent.com/assets/16547949/25400891/fb18941a-29c1-11e7-82b9-35e589443a60.jpg)
 {: .center}
 
-Caption here
+Alexa... please add a new line :-)
```
Comment (ruskj) [inline _posts/0000-01-21-maricay.md]: :+1:

Comment (ruskj) [inline _posts/0000-01-21-maricay.md]: review

Comment (ruskj) [reply _posts/0000-01-21-maricay.md]: test

Comment (maricay) [reply _posts/0000-01-21-maricay.md]: testing

**Comment by maricay** [comment]
merge

**Pull request merged**